packages/content-common/src/types.ts

```diff
@@ -104,6 +104,24 @@ export type DisableEnableSectionApiInput = {
   disabled: boolean;
 };
 
+export type UpdateCustomSectionApiInput = {
+  externalId: string;           // identifier for the Section being updated
+  title?: string;
+  description?: string;
+  heroTitle?: string;
+  heroDescription?: string;
+  startDate?: string;           // YYYY-MM-DD
+  endDate?: string;             // YYYY-MM-DD
+  scheduledSurfaceGuid?: string;
+  iab?: IABMetadata;
+  sort?: number | null;         // allow null to clear
+  // createSource is intentionally NOT updatable for custom sections,
+  // but if present, enforce MANUAL (see resolver).
+  createSource?: ActivitySource;
```

Comment:
lets actually make `createSource` required -- this mutation is to be used via the admin tool & will always be present